tasks/main.yml

```diff
@@ -143,3 +143,30 @@
     name: "{{ fluentd_service_name }}"
     enabled: true
     state: started
+
+- name: Check if pid directory exists
+  stat:
+    path: "{{ fluentd_pid_dir }}"
+  register: register_fluentd_stat_pid_dir
+  changed_when: false
+  when: not ansible_check_mode
+
+- name: Assert pid directory exists
+  assert:
+    that:
+      - register_fluentd_stat_pid_dir.stat.exists
+      - register_fluentd_stat_pid_dir.stat.isdir
+  when: not ansible_check_mode
+
+- name: Check if pid file exists
```

Comment:
some explanation should be written why fluentd fails to create pid file.
restarting fluentd does not solve the problem?